app/views/layouts/devise.html.erb

```diff
@@ -0,0 +1,36 @@
+  <!DOCTYPE html>
+  <html lang='pt'>
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>Easy Lives</title>
+      <%= csrf_meta_tags %>
+      <%= csp_meta_tag %>
+
+      <%= stylesheet_link_tag    'authentication', media: 'all' %>
```

Comment:
```suggestion
      <%= stylesheet_link_tag 'authentication', media: 'all' %>
```

Reply:
@frankyston, tip: you could've just clicked on the above `Commit suggestion` button 🙂

Reply:
Cool, I never got to use the cue button